common/v2/features/AddAccount/AddAccount.tsx

```diff
@@ -275,23 +275,23 @@ const WalletDecrypt = withRouter<Props>(
       return this.WALLETS[selectedWalletKey];
     }
 
-    public handleCreateAccount = async (createAccount: any) => {
+    public handleCreateAccount = (createAccount: any) => {
```

Comment:
Let's type this parameter so we don't accidentally misuse it: `(createAccount: Account => void)` (means a function that takes an `Account` and returns anything or nothing)